modules/openapi-generator/src/main/resources/r/api.mustache

```diff
@@ -173,7 +173,7 @@
       }
     },
 
-    {{{operationId}}}WithHttpInfo = function({{#requiredParams}}{{paramName}}, {{/requiredParams}}{{#optionalParams}}{{paramName}}={{^defaultValue}}NULL{{/defaultValue}}{{{defaultValue}}}, {{/optionalParams}}...){
+    {{{operationId}}}WithHttpInfo = function({{#requiredParams}}{{paramName}}, {{/requiredParams}}{{#optionalParams}}{{paramName}}={{^defaultValue}}NULL{{/defaultValue}}{{{defaultValue}}}, {{/optionalParams}}accept=NULL, ...){
```

Comment:
This could also be made more generic by identifying a headers field, and allowing it to over-ride any header params. It should probably expect a named list just like the httr library already passes to the add_headers method.